components/navigation/ui/src/main/resources/PhenoTips/DatabaseFilters.xml

```diff
@@ -61,6 +61,7 @@
 #if ($activeFields.contains('external_id'))
 * By **external identifier**: {{html clean=false}}&lt;p class="xHint"&gt;(The hospital ID manually entered)&lt;/p&gt;&lt;input type="text" name="external_id" value=""/&gt;{{/html}}
 #end
+* By **privacy level** {{html clean=false}}&lt;br&gt; #foreach ($visibility in $services.permissions.listVisibilityOptions()) &lt;input type="checkbox" class="multipleSelection privacyExport" name="visibility" value="$visibility.value" #if ($visibility != 'hidden')checked#end/&gt;$visibility.label&lt;br&gt;#end&lt;input type="hidden" name="visibility/class" value="PhenoTips.VisibilityClass"/&gt;{{/html}}
```

Comment:
In XHTML it is illegal to have non-closed tags, such as `<br>`. The right syntax is `<br />`.